docs/endpoints/openrtb2/amp.md

```diff
@@ -14,18 +14,73 @@ The only caveat is that AMP BidRequests must contain an `imp` array with one, an
 
 All AMP content must be secure, so this endpoint will enforce that request.imp[0].secure = 1. Saves on publishers forgetting to set this.
 
+### Request
+
+Valid Stored Requests for AMP pages must contain an `imp` array with exactly one element.  It is not necessary to include a `tmax` field in the Stored Request, as Prebid Server will always use the smaller of the AMP default timeout (1000ms) and the value passed via the `timeoutMillis` field of the `amp-ad.rtc-config`.
```

Comment:
In pbs-java _`ap-ad.rtc-config.timeoutMillis`_ is called as _`amp.default-timeout-ms`_